From 91cf9b2caf68a8ad49e10977d41d3a385c0b21be Mon Sep 17 00:00:00 2001
Message-ID: <91cf9b2caf68a8ad49e10977d41d3a385c0b21be.1726926634.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1726926633.git.sharlatanus@gmail.com>
References: <cover.1726926633.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 18 Sep 2024 14:16:15 +0100
Subject: [PATCH v2 11/25] gnu: python-pysiaf: Update to 0.23.3.

* gnu/packages/astronomy.scm (python-pysiaf): Update to 0.23.3.
[propagated-inputs]: Remove python-numpydoc.
[native-inputs]: Add python-setuptools-scm.

Change-Id: Ia20751efc79d391a01b0b50e728646ac38e6b7eb
---
 gnu/packages/astronomy.scm | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index ca521aab0a..dae1fd0a0f 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -5088,18 +5088,18 @@ (define-public python-pyregion
 (define-public python-pysiaf
   (package
     (name "python-pysiaf")
-    (version "0.22.0")
+    (version "0.23.3")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "pysiaf" version))
        (sha256
-        (base32 "08wb98k9k4f04455da5ns9rif8pl9r3ih537w1yj393hkjjiyzfz"))))
+        (base32 "16qbg5n2bw2wr3i8a040i7z7az3w0pn508y6xggy05viwdli6br8"))))
     (build-system pyproject-build-system)
     (arguments
      (list
       #:test-flags
-      #~(list "-n" "auto"
+      #~(list "--numprocesses" "auto"
               ;; Disable 2 failing tests, see
               ;; <https://github.com/spacetelescope/pysiaf/issues/338>
               "-k" (string-append "not test_write_jwst_siaf_xlsx"
@@ -5109,12 +5109,13 @@ (define-public python-pysiaf
            python-lxml
            python-matplotlib
            python-numpy
-           python-numpydoc
            python-openpyxl
            python-requests
            python-scipy))
     (native-inputs
-     (list python-pytest python-pytest-xdist))
+     (list python-pytest
+           python-pytest-xdist
+           python-setuptools-scm))
     (home-page "https://pysiaf.readthedocs.io/")
     (synopsis "Handling SIAF for space telescopes")
     (description
-- 
2.46.0

